data/xml/2020.nlpcovid19.xml

```diff
@@ -551,8 +551,11 @@
       <author><first>Jaida</first><last>Alsuhaibani</last></author>
       <author><first>Fawziah</first><last>Aljabli</last></author>
       <abstract>The coronavirus disease of 2019 (COVID-19) has a huge impact on economies and societies around the world. While governments are taking extreme measures to reduce the spread of the virus, people are getting affected by these new measures. With restrictions like lockdown and social distancing, it became important to understand the emotional response of the public towards the pandemic. In this paper, we study the reaction of Saudi Arabia citizens towards the pandemic. We utilize a collection of Arabic tweets that were sent during 2020, primarily through hashtags that were originated from Saudi Arabia. Our results showed that people had kept a positive reaction towards the pandemic. This positive reaction was at its highest at the beginning of the COVID-19 crisis and started to decline as time passes. Overall, the results showed that people were so supportive of each other through this pandemic. This research can help researchers and policymakers in understanding the emotional effect of a pandemic on societies.</abstract>
-      <url hash="f25a00f9">2020.nlpcovid19-2.24</url>
+      <url hash="2f7fda9c">2020.nlpcovid19-2.24</url>
       <doi>10.18653/v1/2020.nlpcovid19-2.24</doi>
+      <revision id="1" href="2020.nlpcovid19-2.24v1" hash="f25a00f9"/>
+      <revision id="2" href="2020.nlpcovid19-2.24v2" hash="74cce4ec" date="2021-06-03">changed section 5.3 and figure 3, added section 6</revision>
+      <revision id="3" href="2020.nlpcovid19-2.24v3" hash="2f7fda9c" date="2021-06-03">changed section 5.3 and figure 3, added section 6.</revision>
     </paper>
```

Comment:
It looks like this one got added twice?

Reply:
Yes, because I made a mistake linking the new pdf the first time

Reply:
Okay—we should delete r3 and correct r2 manually, then. Or just reset the XML here to the original and re-process the revision.